src/themes/masschallenge/elements/header.overrides

```diff
@@ -0,0 +1,7 @@
+.ui.normal.header {
+    font-weight: normal;
+}
+
+.ui.dark.header {
+    color: @darkTextColor;
+}
```

Comment:
This may not be necessary if the `header` is refactored per: https://github.com/masschallenge/front-end/pull/183#discussion_r338028957 (unless this color is reused in another location).